.github/workflows/android-ci.yml

```diff
@@ -25,17 +25,11 @@ on:
 
 jobs:
   build:
-    runs-on: ubuntu-20.04
+    runs-on: ubuntu-22.04
     defaults:
       run:
         working-directory: platform/android
-    container:
-      image: ghcr.io/maplibre/android-ndk-r21b
-      credentials:
-        username: ${{ github.repository_owner }}
-        password: ${{ secrets.CR_PAT }}
```

Comment:
What is our plan with this? Should we simply stop publishing this if this approach works out?

Reply:
Nevermind. That is what you did anyway. :D